### DISCUSSION TITLE
Ayuda: Diseño del Pokedex

### DISCUSSION THREAD

**Discussion by oraliahdz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#394] [Q&A]
Hola compañeros, quisiera la opinión de los que están un poco más experimientados en la parte de Frontend
Soy nueva en la parte de CSS, así que tengo una duda. Que es más eficiente, diseñar el Pokedex en Imagen y luego ponerla en mi HTML o hacerlo y diseñarlo todo con CSS (asumiendo que de milagro pueda lograrlo)? 
Me preocupa un poco el efecto de distorción dependiendo del tamaño de la ventana.

**Comment by hackit92** [comment]
Yo he pensado en usar svg para el fondo y hacerlo todo en un grid, de tal forma que pueda hacerla responsiva pero sigo pensando, creo que dejar jpegs fuera sería lo mejor

**Comment by JuanSalvador73** [comment]
Generalmente los pasos a seguir son:

1.- Diseñar el sitio solo en imagen plana, con formato png, jpg, etc.
2.- Ya con la imagen hecha, construyes el HTML, solo el esqueleto con etiquetas en tag sin estilos.
3.- Teniendo el HTML listo, ahora si le pones los estilos siguiendo como guía la imagen plana.

Claro que cuando creas tu imagen desde 0, es porque ya se tiene una idea de como va a quedar el sitio finalmente, sino todo el texto completo, que tipo de info irá.
Hay quienes con mas experiencia crean el sitio sin hacer imagen. Este tuto puede ayudarte:

https://www.youtube.com/watch?v=8-RC-Q7Wtzc

**Comment by Ingrzs** [comment]
como tu puedas y desees, cada quien va a su ritmo lo importante es aprender.
Yo usare una imagen png con fondos transparentes y ahi dentro les pondre mis elementos html

**Comment by ChristianDeM** [comment]
En mi caso lo que hice fue hacer un div central  de la pokedex. luego lo dividi en 2 La parte de arriba es la figura de la pokedex y ya en la parte de abajo  le puse informacion como si fuera una pagina web normal. tratare de darle otro diseño ya que hay cosas que no se ven  del todo te dejo una captura. 
![alt text](https://raw.githubusercontent.com/ChristianDeM/pokedex-demo/main/demo.png)

Tambien puedes hacer otro diseño de pokedex hay muchos diseños dependiendo de cada generacion.

![alt text](https://static.wikia.nocookie.net/b8fb60cf-4a21-42c9-ad33-11e6b9c54d8a/scale-to-width/755)
